src/pages/connections/farmlands.mdx

```diff
@@ -10,43 +10,43 @@ In partnership with Centrapay, Farmlands has two new digital solutions that enab
 Your business can either fully integrate the Farmlands solution to your retail point of sale system, or access a secure web portal when accepting Farmlands Card as a form of payment for a sale anytime, anywhere.
 
 <center>
-```mermaid
-%%{
-  init: {
-    'theme': 'base',
-    'themeVariables': {
-      'secondaryColor': '#f8f8f8',
-      'tertiaryColor': '#f8f8f8'
-    }
-  }
-}%%
+  <div style="max-width: 800px">
+    ```mermaid
+    %%{
+      init: {
+        'theme': 'base',
+        'themeVariables': {
+          'secondaryColor': '#f8f8f8',
+          'tertiaryColor': '#f8f8f8'
+        }
+      }
+    }%%
 
-flowchart TD
-  subgraph F[Farmlands Cardholder]
-    subgraph padding1 [ ]
-      Card(Farmlands Card)
-    end
-  end
+    flowchart TD
+      subgraph F[Farmlands Cardholder]
+        subgraph padding1 [ ]
+          Card(Farmlands Card)
+        end
+      end
 
-  subgraph P[Farmlands Card Partner]
-    subgraph padding2 [Authorisations via: ]
-      subgraph padding3 [ ]
```

Comment:
@alexBinns , I removed the padding here because the X axis padding was too much.
I don't think the added space you get between "Authorizations via:" and the next graph is as big a deal.

Reply:
Yea looking at it I'm not too bothered as its not really optimal either way